### PR TITLE
fix(sentry): take out switching of env in each interval

### PR DESCRIPTION
**Pull request by stebenz**

### CODE REVIEW

File: operator/zitadel/kinds/iam/zitadel/adapt.go [hunk 1]
```diff
@@ -55,15 +52,17 @@ func AdaptFunc(
...
+			_, _, sendAnalytics := mntr.Environment()
+			if sendAnalytics &&
+				desiredKind != nil &&
+				desiredKind.Spec != nil &&
+				desiredKind.Spec.Configuration != nil &&
+				desiredKind.Spec.Configuration.DNS != nil &&
+				desiredKind.Spec.Configuration.DNS.Domain != "" {
+				monitor.SwitchEnvironment(strings.ToLower(strings.ReplaceAll(desiredKind.Spec.Configuration.DNS.Domain, ".", "-")))
+			}
+		*/
```
Comment (livio-a) [inline operator/zitadel/kinds/iam/zitadel/adapt.go:65]: please remove part or uncomment again

**Comment by eliobischof** [comment]
implemented in https://github.com/caos/orbos/pull/933 so it is called once instead of never

- [ ] `go get github.com/caos/orbos` as soon as it Is meged

**Review by eliobischof — APPROVED** [review]

**Review by eliobischof — APPROVED** [review]

**Comment by eliobischof** [comment]
@stebenz close?

**Comment by eliobischof** [comment]
not needed anymore, as v2 is released

**Pull request closed without merging**